### PR TITLE
Add new google tag manager snippets

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
https://github.com/ualbertalib/library-cms/issues/561

Add snippets for new google tag manager

### CODE REVIEW

**Review by pgwillia — COMMENTED** [review]
Looks okay to me.  Do we care about the readability?  Noticed the gtag script is laid out in a much more readable way.

File: app/views/layouts/2020.html.erb [hunk 1]
```diff
@@ -9,26 +9,41 @@
...
+      gtag('config', 'G-5QT07E47HT');
+    </script>
+    <!-- Google Tag Manager -->
+    <script>
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-PVV4HQM4');
+    </script>
```
Comment (pgwillia) [inline app/views/layouts/2020.html.erb:26]:
```suggestion
    <script>
      (
        function(w, d, s, l, i) { 
          w [l] = w [l] || [];
          w [l].push(
              { 'gtm.start': new Date().getTime(),
              event :'gtm.js' }
          );
          var f = d.getElementsByTagName(s) [0],
          j = d.createElement(s),
          dl = l != 'dataLayer' ? '&l=' + l :'';
          j.async = true;
          j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
          f.parentNode.insertBefore(j, f);
        }
      )(window, document, 'script', 'dataLayer', 'GTM-PVV4HQM4');
    </script>
```

Comment (pgwillia) [reply app/views/layouts/2020.html.erb:26]: @ConnorSheremeta do you think this is an improvement?  If not we can go back to the condensed code.

Comment (ConnorSheremeta) [reply app/views/layouts/2020.html.erb]: I think there are some arguments to be made to keep it condensed and obfuscated, but I think in this case, both to match what was done with the gtag script above and considering it is a relatively short script, this change would be an improvement :+1:

**Review by jefferya — APPROVED** [review]
LGTM!

I'm curious as to why the Google Tag Manager is not added to other views, however, this satisfies the original issue scope so LGTM!

**Comment by ConnorSheremeta** [comment]
> LGTM!
> 
> I'm curious as to why the Google Tag Manager is not added to other views, however, this satisfies the original issue scope so LGTM!

library-cms/app/views/layouts/2020.html.erb is the overarching layout for the main content of the cms (some instances where views do not use this layout but they are more-or-less considered a separate site.) Also added Google Tag Manager to app/views/layouts/2020_bootstrap_v5.html.erb as pages will be migrated to use that layout over time.

**Pull request merged**